sheet2linkml/cli.py

```diff
@@ -49,30 +57,55 @@
     default=True,
     help="Controls whether we use the CCDH Terminology Service to add enumerated values for attributes.",
 )
-def main(output, filter_entity, logging_config, write_mappings, include_terminologies):
+def main(
+    google_sheet_id,
+    output,
+    filter_entity,
+    logging_config,
+    write_mappings,
+    include_terminologies,
+):
     # Display INFO log entry and up.
-    logging.config.fileConfig(logging_config)
+    if os.path.exists(logging_config):
+        logging.config.fileConfig(logging_config)
```

Comment:
This is a great handler. To configure the logger only if a `logging.ini` file is provided.

One quick nit I want to suggest here is that, we should probably remove the `default="logging.ini"` line on [L#48](https://github.com/cancerDHC/sheet2linkml/blob/add-dummy-sheet-test/sheet2linkml/cli.py#L48) since it defaults to that if no file is provided, and the if condition will always be skipped.

Since you mention you want to log `INFO` level and up, you can also configure `logging` in the `else` part of the block like such:

```
# if block
else:
    logging.basicConfig(level=logging.INFO)
```

Reply:
Fixed in c6f6e30! Note that it's not recommended to use a link like https://github.com/cancerDHC/sheet2linkml/blob/add-dummy-sheet-test/sheet2linkml/cli.py#L48 that refers to a branch (`add-dummy-sheet-test`) instead of a particular commit, since line 48 on that file might change if the file changes. The recommendation is to find the line you want and then [hit the `y` key](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files#press-y-to-permalink-to-a-file-in-a-specific-commit) to link to a particular commit.